doc/proxy-clients.rst

```diff
@@ -91,6 +91,11 @@ include that port in the base URL::
 
     $varnish = new Varnish($servers, 'my-cool-app.com:8080');
 
+Another optional parameter available on Varnish client is ``tagsHeader``, which allows you to
+change the default HTTP header used for tags, ``X-Cache-Tags``::
```

Comment:
lets say "tagging" instead of "tags", and link that to the response tagger documentation.